### DISCUSSION TITLE
simulator.html

### DISCUSSION THREAD

**Discussion by youbecha** [aqualinkd/AqualinkD#191] [Q&A]
saw the link in the other forum...opened it up in my working AqualinkD and see something...but hard to view.

Is there any more information on this portion of AqualinkD...is it dead?

My OneTouch died, and I am using AqualinkD to replace it...but it would be nice to be able to edit the schedule.

**Comment by sfeakes** [comment]
The simulator has a very low priority and needs a lot of changes for it to be effective.  I would suggest deleting the schedule from the controller and using your home automation hub (or chron) to do the scheduling through aqualinkd
